<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show</class>
 <widget class="QWidget" name="Show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RES.qrc">
    <normaloff>:/icon/office-reports-icon-outline-office-reports-vector-icon-for-web-design-isolated-on-white-background-700-219898713.jpg</normaloff>:/icon/office-reports-icon-outline-office-reports-vector-icon-for-web-design-isolated-on-white-background-700-219898713.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="2">
    <widget class="QLabel" name="show_album">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="infor_id">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="buddy">
      <cstring>infor_id</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="show_year">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="infor_lng">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="infor_writer">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Writer:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="show_writer">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Song information</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="show_time">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="infor_genre">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Genre:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="show_artist">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="infor_album">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Album:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="show_lng">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="show_id">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>show_id</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLabel" name="show_genre">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="infor_artist">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Artist:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="infor_year">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Relase year:</string>
     </property>
     <property name="buddy">
      <cstring>infor_year</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="infor_time">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Duration:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="show_name">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Modern No. 20</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RES.qrc"/>
 </resources>
 <connections/>
</ui>
